resources/versioning.md

```diff
@@ -0,0 +1,92 @@
+
+But there is a price for that. When we make that cut, we reduce the number of users who can adopt our SDK/service.
+
+Currently, all supported versions are set in the CI of each repository.
+
+This is the list of minimum supported language version runtimes*:
+
+| Language/Framework | Version |
+| ------------------ | ------- |
+| .NET | netstandard2.0 |
+| Dart | >= 2.12.0 |
+| Go | >= 1.16 |
+| Java | >= 8.0 |
+| JavaScript/Node | >= 14 |
+| Swift | >= 5.2 |
+| PHP | >= 7.4 |
+| Python | >= 3.7 |
+| Ruby | >= 2.6 |
+| Rust | N/A |
+
+When an SDK drops a minimum required version of the runtime, we release a new MAJOR version of the SDK.
+
+\* This list may need to be updated. Always check the current version in the integration's repository.
+
+## Correlation between Meilisearch engine versioning and the SDKs
+
+SDKs do not follow the Meilisearch engine version. It means that there is no correlation between SDK's version and Meilisearch; an SDK may be in version `v10.0.0` while the Meilisearch engine is in `v1.0.0`.
+
+Suppose the version of an SDK is the same as the latest version of Meilisearch. It is just a happy coincidence.
+
+A major release in Meilisearch may also not lead to a major release in the SDKs. For example, if we get a Meilisearch v2 in the future, there is no guarantee that the Meilisearch Dart SDK will update from v10 to v11.
+
+You can find out which version of the Meilisearch engine an SDK is compatible with by going to the `Compatibility with Meilisearch` section of every SDK README page. For example, the [README](https://github.com/meilisearch/meilisearch-ruby/#-compatibility-with-meilisearch) page of the ruby SDK.
+
+To illustrate, the idea is to have a similar table in every repository.
+
+| Meilisearch engine version | Integration version |
+| -------------------------- | ------------------- |
+| v0.30 | v0.14.0 |
+| v1.0 | >= v0.15.0 |
+| v1.1 | v0.18.0 |
+
+Given the example above, it means that the integration in v0.15 works with Meilisearch v1.1, but may only some of the new features introduced in v1.1 will work with this integration version.
+
+We're still working on making all of our SDKs stable, but first, we would like to evolve some questions, like having more feedback about the public API we expose to the users. We're eager to receive feedback on changes you'd like to see in some SDKs before their v1 release.
+
+## When to we update SDKs to v1?
+
+We are getting closer to a possible v1 in the SDKs, but there has yet to be a real roadmap.
+If you have a specific use case where you need the SDKs in v1, please let us know through the issues.
+
+## Important facts
+
+:warning: Always check the Meilisearch engine version you are running and the version supported by your current SDK version.
+
+:warning: If you use a `raw` method like `GetAllRawIndexesAsync`, `allRaw`, `rawSearch`, and similar. Please double-check the Meilisearch version you're running.
```

Comment:
Is there still some `all` like `getAll` somewhere in some SDK?

Reply:
I would have to check that, but they just have a bad naming they are not so harmful like the `raw` ones.